andela/ah-frontend-targaryen
# #161717199 adding images to articles

#### What does this PR do?

- Allow all users to add images while creating articles
- Allows all users to view a specific article
- Allows users to view the articles they have authored
#### Description of Task to be completed?

Currently, our application has no capability to add images to articles and viewing specific articles

This PR allows us to add images and view specific articles


#### How should this be manually tested?

- Run the application `npm start`
- Login and on the dashboard click `Let’s write`
- Add the necessary details and click the `image icon` in the toolbar to add an image
- Save changes and navigate back to the dashboard. 
- The newly created article should be top of the articles displayed, click `Read More`
- You are now viewing your article and it’s image

#### What are the relevant pivotal tracker stories?

[#161717199](https://www.pivotaltracker.com/story/show/161717199)



#### Any background context you want to add?
N/A



#### Screenshots
N/A

## Checklist:
